clients/src/main/java/nl/beyco/webserver/dto/Coffee.java

```diff
@@ -26,21 +35,23 @@
     @NotBlank(message = "sector is mandatory")
     private String sector;
 
-    @NotBlank(message = "harvestAt is mandatory")
-    private String harvestAt;
+    @NotNull(message = "harvestAt is mandatory")
+    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
+    @JsonSerialize(using = LocalDateTimeSerializer.class)
+    private LocalDateTime harvestAt;
 
     @NotBlank(message = "qualitySegment is mandatory")
     private String qualitySegment;
 
-    @NotBlank(message = "Minimal screen size is mandatory")
-    private String minScreenSize;
+    @Min(0)
+    private int minScreenSize;
 
-    @NotBlank(message = "Maximal screen size is mandatory")
-    private String maxScreenSize;
+    @Min(0)
```

Comment:
^ en min mag niet hoger zijn dan max natuurlijk

Reply:
Dit is niet af te vangen met javax validatie. Dus dan voeg ik het in de blockchain toe.